packages/core/src/components/Dropdown/dropdownClasses.tsx

```diff
@@ -0,0 +1,54 @@
+import { getClasses } from "utils";
```

Comment:
Maybe it would be better to separate the classes for the dropdown and dropdown list in different files to maintain consistency 🙏 That's what we did with the AppSwitcher for example.